frontend/src/mocks-server/__mockdata__/licenses.ts

```diff
@@ -0,0 +1,90 @@
+export const licensesSuccessResponse = {
+	status: 'success',
+	data: {
+		trialStart: 1695992049,
+		trialEnd: 1697806449,
+		onTrial: true,
+		workSpaceBlock: false,
+		trialConvertedToSubscription: false,
+		gracePeriodEnd: -1,
+		licenses: [
+			{
+				key: '9d877476-a092-4c62-b010-10453c9a5e41',
```

Comment:
Can we remove the key & activationId and replace with dummy texts?

Reply:
Okay